docs/design/coreclr/botr/clr-abi.md

```diff
@@ -322,33 +322,11 @@ Finally1:
 
 Note that JIT64 does not implement this properly. The C# compiler used to always insert all necessary "step" blocks. The Roslyn C# compiler at one point did not, but then was changed to once again insert them.
 
-## The PSPSym and funclet parameters
+## Funclet parameters
```

Comment:
@dotnet/jit-contrib Are you ok with deleting the outdated information about the ABI used by earlier .NET versions from this doc?

Reply:
(I was thinking to keep it in some form as long as the R2R compatibility is retained.)

Reply:
> R2R compatibility is retained

We have taken a hard break for .NET 10 for other reasons already, so we do not really need to worry about R2R compatibility with this one.

Reply:
> @dotnet/jit-contrib Are you ok with deleting the outdated information about the ABI used by earlier .NET versions from this doc?

IMO, yes. I would expect to need to find the matching document in the appropriate servicing source tree to understand a previous ABI, rather than have the latest document cover all possible histories of the ABI.